docs/dev_guide/contents/dependencies.rst

```diff
@@ -0,0 +1,22 @@
+.. _dependency_versions:
+
+*************************
+Dependency Support Policy
+*************************
+
+.. note::
+
+    This policy is based on `NEP-0029`_.
+
+sunpy has a short list of core dependencies (Python, numpy, astropy, parfive) and a long list of optional dependencies.
+The minimum version of these packages that we enforce follows this policy.
+
+* Python: Released in the prior 42 months from the anticipated release date.
+* Astropy: Released in the prior 12 months from the anticipated release date.
+* Everything else: Released in the prior 24 months from the anticipated release date.
+
+There are two caveats here: parfive and drms as they are packages we control and might need to bump (especially drms as it's so closely coupled to JSOC).
```

Comment:
Do we really need to do this? I think we reserve the right to bump bugfix (a.b.x > a.b.x+1) requirements for dependencies whenever(?), which should cover parfive and drms fixes.

Reply:
I will remove this.